.github/workflows/main.yml

```diff
@@ -7,15 +7,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node: ['12.x', '14.x']
+        node: ['15.x']
```

Comment:
Why 15 and not 14?

Reply:
15 includes npm 7 by default, as the GItHub action doesn't provide a way to install a custom version of npm it's the simplest setup.

Reply:
I'm mostly just wondering if we might run into issues here because we are not actively testing with node 14?

Reply:
I don't think the node version matters much here as anyways the IDX library targets both node and browsers, it's compiled to support older versions.
It could affect the tools library and the CLI but unless we're explicitly using APIs from v15 or dependencies using them, I don't think we would have any issue.